src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: $nav-hight;
```

Comment:
It appears there is a typo in the variable name `$nav-hight`. According to the code style guide, variable names should be spelled correctly. You probably meant `$nav-height`.